src/data/step.puml

```diff
@@ -0,0 +1,36 @@
+@startuml
+skinparam activity {
+BackgroundColor #76CDD8
+ArrowColor black
+ArrowFontSize 13
+ArrowFontStyle bold
+ArrowFontName Segoe UI
+BorderColor black
+FontName Segoe UI
+FontSize 15
+}
+(*) --> "handle Step"
```

Comment:
I'd add here a label: `-->[step]` to indicate that we got a step as parameter

Reply:
Should we apply this approach on other files with parameters as well?

Reply:
That would be great :)